bindings/rust/standard/integration/src/features/pq.rs

```diff
@@ -20,7 +20,7 @@ pub async fn get_streams() -> Result<(TcpStream, TcpStream), tokio::io::Error> {
 }
 
 #[test_log::test(tokio::test)]
-async fn s2n_client() -> Result<(), Box<dyn std::error::Error>> {
+async fn s2n_client_mldsa() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Nit: I'd probably invert the naming for all your tests. "s2n_mldsa_client" instead.